_layouts/base.html

```diff
@@ -3,17 +3,16 @@
 <!--
     ============================================================================
 
-    Hey! If you're viewing this, you should probably come work on our Technology
-    & Innovation team. We're always looking for a few great designers,
-    developers, data scientists, and network, infrastructure, privacy and
-    security pros. Keep an eye on our job opportunities at:
-    http://www.consumerfinance.gov/jobs/
+    Hey! If you're viewing this, you should probably come work on our
+    Technology & Innovation team. We're always looking for a few great
+    designers; developers; data scientists; and network, infrastructure,
```

Comment:
That's cool. Then I'd recommend removing the words "a few", as we're listing out a lot of roles.

On "geeks": It's certainly possible. I was having issues coming up with other nouns to describe tech folks.